realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -151,6 +159,30 @@ RealmProxyMediator getSchemaMediator() {
     }
 
     /**
+     * Indicates if there is available asset file for copy action.
+     *
+     * @return true if there is asset file, false otherwise.
+     */
+    boolean hasAssetFile() {
+        return !TextUtils.isEmpty(assetFilePath);
+    }
+
+    /**
+     * Returns input stream object to the Realm asset file.
+     *
+     * @return input stream to the asset file.
+     * @throws IOException if copying the file fails.
+     */
+    InputStream getAssetFile() throws IOException {
+        Context context = contextWeakRef.get();
+        if (context != null) {
+            return context.getAssets().open(assetFilePath);
+        } else {
+            throw new IllegalArgumentException("Context can not be null. Use application context instead.");
```

Comment:
I would specify this a bit more. Something like 
"Context should not be null. Use Application Context instead of Activity Context."